docs/docs/installation/rasa-pro/installation.mdx

```diff
@@ -19,6 +19,9 @@ To learn more about deployments, please visit the following sections:
 - [Deploying Rasa Plus in production](../../deploy/deploy-rasa-plus)
 - [Deploying Rasa Pro Services in production](../../deploy/deploy-rasa-pro-services)
 
+To learn more about which version of Rasa Pro Services is compatible with Rasa Plus, please head over to the
+[compatibility matrix](../../compatibility-matrix.mdx).
```

Comment:
not sure if it's the best place for that plug. LMK!

Reply:
I think the place is right